sdcm/cluster.py

```diff
@@ -2553,7 +2553,7 @@ def update_scylla_packages(node, queue):
             node.remoter.run('sudo yum update -y --skip-broken')
```

Comment:
Need to remove this since what happens is that when using older AMI, it updates to latest available Scylla.
That makes conflicts when using custom RPMs from older versions